frontend/src/triggerAudioVisuals.js

```diff
@@ -16,7 +16,7 @@ const triggerAudioVisuals = (quantisedTubeData, instruments, arrivalEffectsToggl
     const note = instruments.noteAssignFunctions[train.lineName](train.stationName);
     TIMEOUTS.setTimeout(() => {
       const now = Tone.now(); // the audio context time
-      const minVelocity = 0.5;
+      const minVelocity = 0.8;
```

Comment:
Nice, this actually narrows the dynamic range (less variation), since the max is always 1, and so what we are setting here is the minimum value below that. But if this sounded best to you, I can dig it!